cli/deno_dir.rs

```diff
@@ -1618,299 +1576,240 @@ mod tests {
-    let referrer = referrer_.to_str().unwrap();
-
-    let expected_module_name = "https://unpkg.com/util";
-    let expected_filename =
-      normalize_to_str(deno_dir.deps_https.join("unpkg.com/util").as_ref());
-
-    let (module_name, filename) =
-      deno_dir.resolve_module(specifier, referrer).unwrap();
-    assert_eq!(module_name, expected_module_name);
-    assert_eq!(filename.to_str().unwrap().to_string(), expected_filename);
-  }
-
-  #[test]
-  fn test_resolve_module_referrer_dot() {
-    let (_temp_dir, deno_dir) = test_setup();
-
-    let specifier = "tests/001_hello.js";
-
-    let cwd = std::env::current_dir().unwrap();
-    let expected_path = cwd.join(specifier);
-    let expected_module_name =
-      Url::from_file_path(&expected_path).unwrap().to_string();
-    let expected_filename = normalize_to_str(&expected_path);
-
-    let (module_name, filename) =
-      deno_dir.resolve_module(specifier, ".").unwrap();
-    assert_eq!(module_name, expected_module_name);
-    assert_eq!(filename.to_str().unwrap().to_string(), expected_filename);
-
-    let (module_name, filename) =
-      deno_dir.resolve_module(specifier, "./").unwrap();
-    assert_eq!(module_name, expected_module_name);
+    let filename = deno_dir.url_to_deps_path(&specifier).unwrap();
     assert_eq!(filename.to_str().unwrap().to_string(), expected_filename);
   }
 
-  #[test]
-  fn test_resolve_module_referrer_dotdot() {
-    let (_temp_dir, deno_dir) = test_setup();
-
-    let specifier = "tests/001_hello.js";
-
-    let cwd = std::env::current_dir().unwrap();
-    let expected_path = cwd.join("..").join(specifier);
-    let expected_module_name =
-      Url::from_file_path(&expected_path).unwrap().to_string();
-    let expected_filename = normalize_to_str(&expected_path);
-
-    let (module_name, filename) =
-      deno_dir.resolve_module(specifier, "..").unwrap();
-    assert_eq!(module_name, expected_module_name);
-    assert_eq!(filename.to_str().unwrap().to_string(), expected_filename);
-
-    let (module_name, filename) =
-      deno_dir.resolve_module(specifier, "../").unwrap();
-    assert_eq!(module_name, expected_module_name);
-    assert_eq!(filename.to_str().unwrap().to_string(), expected_filename);
-  }
+  // TODO(bartlomieju): commented tests should be ported to `url_to_deps_path`
+  //  #[test]
+  //  fn test_resolve_module_3() {
+  //    let (_temp_dir, deno_dir) = test_setup();
+  //
+  //    let specifier_ =
+  //      deno_dir.deps_http.join("unpkg.com/liltest@0.0.5/index.ts");
+  //    let specifier = specifier_.to_str().unwrap();
+  //    let referrer = ".";
+  //
+  //    let expected_module_name = "http://unpkg.com/liltest@0.0.5/index.ts";
+  //    let expected_filename = normalize_to_str(
+  //      deno_dir
+  //        .deps_http
+  //        .join("unpkg.com/liltest@0.0.5/index.ts")
+  //        .as_ref(),
+  //    );
+  //
+  //    let (module_name, filename) =
+  //      deno_dir.resolve_module(specifier, referrer).unwrap();
```

Comment:
@ry do you remember why `resolve_module` was resolving both:
- remote Urls to deps filepath
- deps filepath to remote Url?

I managed to completely remove this method without breaking anything

Reply:
I don't remember - and it's probably of questionable design anyway. If it's passing tests I'm happy.